templates/threatconnect.hbs

```diff
@@ -982,56 +1088,183 @@
+                            >
+                              {{#each caseAttributeTypes.data as |attribute|}}
+                                <option value="{{json-stringify attribute}}">
+                                  {{attribute.name}}
+                                </option>
+                              {{/each}}
+                            </select>
+                          </div>
+                        {{/if}}
                       </div>
+
+                      {{#if case.attributes.data.length}}
+                        <div class="info-bar-pill" style="width: 100%; padding-bottom: 5px;">
+                          <span class="info-bar-key">Case Attributes</span>
+                          {{#each case.attributes.data as |caseAttr|}}
+                            <div class="info-bar-value">
+                              <div><strong>{{caseAttr.type}}</strong></div>
+                              <div>{{caseAttr.value}}</div>
+                            </div>
+                          {{/each}}
+                        </div>
+                      {{else}}
+                        <span class="info-bar-key" style="margin-top: 10px; margin-bottom: 10px;">No Attributes
+                          Associated</span>
+                      {{/if}}
+
+                      {{#if case.__isEditing}}
+                        {{#if case.__newAttributes}}
+                          <div class="info-bar-pill" style="width: 100%; padding-bottom: 5px;">
+                            <span class="info-bar-key">New Case Attributes</span>
+                            {{#each case.__newAttributes as |caseAttr index|}}
+                              <div style="display: flex; justify-content: space-between;">
+                                <div class="info-bar-value">
+                                  <div><strong>{{caseAttr.type}}</strong></div>
+                                  <div>{{caseAttr.value}}</div>
+                                </div>
+                                <button
+                                  class="remove-attribute"
+                                  {{on "click" (fn (action "removeAttribute" case.id orgData.indicator.id index))}}
+                                >
+                                  {{fa-icon icon="times"}}
+                                </button>
+                              </div>
+                            {{/each}}
+                          </div>
+                        {{/if}}
+                      {{/if}}
+
+                      {{#if block.userOptions.enableEditingCases}}
+                        <div>
+                          <button
+                            class="btn btn-secondary btn-add-tag"
+                            style="border-radius: 0px;"
+                            {{action "toggleEdit" case.id orgData.indicator.id}}
+                          >
+                            {{if case.__isEditing "Save" "Edit"}}
```

Comment:
Just like the create case button we need to disable this button while we're saving so the user doesn't click on it multiple times.  In addition, we need some visible feedback that the integration is doing something when the button is clicked.  I would add in the spin icon here while the saving is happening.  Something like:

```
{{#if case.__isSaving}}
   {{fa-icon icon="spinner-third" fixedWidth=true spin=true}}
{{else if case.__isEditing}}
   Save
{{else}}
   Edit
{{/if}}